src/Models/BaseElement.php

```diff
@@ -569,8 +581,8 @@ public function forTemplate($holder = true)
         $config = SiteConfig::current_site_config();
 
         if ($config->Theme) {
-            Config::inst()->update('SSViewer', 'theme_enabled', true);
-            Config::inst()->update('SSViewer', 'theme', $config->Theme);
+            Config::inst()->update(SSViewer::class, 'theme_enabled', true);
+            Config::inst()->update(SSViewer::class, 'theme', $config->Theme);
```

Comment:
We should use `Config::modify()->set()` here as this API is deprecated.

Also the `theme` config property is deprecated, so we should instead push a theme to the `themes` array instead: `Config::modify()->merge(SSViewer::class, 'themes', [$config->Theme]);`